### ISSUE TITLE
How to fine-tune for multi-label classification tasks

### ISSUE THREAD

**Issue by Bentonmaster** [2U1/Qwen2-VL-Finetune#209]
Thank you for your outstanding work
I would like to know whether your fine-tuning framework supports training and fine-tuning for multi-label image classification tasks.
If possible, could you provide an example of constructing a dataset?

**Comment by 2U1** [comment]
The code is not for multi-label so you need to tweak a bit in the `cls_dataset.py`.

**Comment by 2U1** [comment]
@BingfengHan @Bentonmaster 
For multi-label classification, you should change the label part in the `cls_dataset.py`.
The dataset format should look like
```
"label": [
class1: 0
class2: 1
class3: 1
]
```
It's just an example so you could make it similar to this.
